esp/logging/logginglib/logthread.cpp

```diff
@@ -853,9 +853,15 @@ bool CLogRequestReader::readLogRequestsFromTankFile(const char* fileName, String
         if (!logSerializer.readALogLine(fileIO, finger, data))
             break;
 
+        bool skipLogRequest = false;
         StringBuffer GUID, logRequest;
-        if (!parseLogRequest(data, GUID, logRequest))
-            IERRLOG("Invalid logging request in %s", fileName);
+        if (!parseLogRequest(data, GUID, logRequest, skipLogRequest))
+        {
+            if (skipLogRequest)
+                ESPLOG(LogMax, "#### Skip: %s", GUID.str());
```

Comment:
If the skip must be logged, would it be helpful for the log message to include more context than just the GUID? We will have logged individual log variants that were disabled, but without the GUID. The recorded TID can probably be used to associate the disabled entries with the skipped agent, but that seems like an unnecessary complication.